ddp-workspace/projects/ddp-osteo/src/app/components/workflow-progress/workflow-progress.component.ts

```diff
@@ -13,6 +13,7 @@ export class WorkflowProgressComponent implements OnChanges {
   @Input() public currentActivity: string;
```

Comment:
Would you mind annotating with what this attribute means? Is this the activity instance guid or activity  guid or something else? (I think it is the Activity GUID) but needed to do some digging...